### DISCUSSION TITLE
<title>I cant seem to figure out why the site is getting 404

### DISCUSSION THREAD

**Discussion by TJ-Amici** [community/community#46767] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am new to the whole pages thing so I have no doubt that its me doing something wrong but I cant see it. It says published and everything went OK.

**Comment by radar-blip** [comment]
I think its because the index.html is inside a folder, and GitHub Pages can't see it.
Put index.html at root and see if that works.
If it does, you will have to adjust your links accordingly.

**Reply by TJ-Amici** [reply]
Thanks, it worked but I can't find the path to get the images to show any thoughts.

**Reply by radar-blip** [reply]
It looks like your images are showing, maybe it just took some time.
